vault/token_store.go

```diff
@@ -2172,7 +2172,7 @@ func (ts *TokenStore) tokenStoreRoleCreateUpdate(
 				resp = &logical.Response{}
 			}
 			resp.AddWarning(fmt.Sprintf(
-				"Given explicit max TTL of %d is greater than system/mount allowed value of %d seconds; until this is fixed attempting to create tokens against this role will result in an error",
```

Comment:
We actually want this to be an integer, not a float, so it should probably stay `%d` but the parameter be cast to an `int64`.